R/filter_confirmation.R

```diff
@@ -121,6 +185,45 @@
 #'   filter = AVALC == "CR" & all(AVALC.join %in% c("CR", "NE")) &
 #'     count_vals(var = AVALC.join, val = "NE") <= 1
 #' )
+#'
+#' # select observations with AVALC == "PR", AVALC == "CR" or AVALC == "PR"
+#' # at a subsequent visit at least 20 days later, only "CR", "PR", or "NE"
+#' # in between, at most one "NE" in between, and "CR" is not followed by "PR"
+#' data <- tibble::tribble(
+#'   ~USUBJID, ~ADY, ~AVALC,
+#'   "1",         6, "PR",
+#'   "1",        12, "CR",
+#'   "1",        24, "NE",
+#'   "1",        32, "CR",
+#'   "1",        48, "PR",
+#'   "2",         3, "PR",
+#'   "2",        21, "CR",
+#'   "2",        33, "PR",
+#'   "3",        11, "PR",
+#'   "4",         7, "PR",
+#'   "4",        12, "NE",
+#'   "4",        24, "NE",
+#'   "4",        32, "PR",
+#'   "4",        55, "PR"
+#' )
+#'
+#' filter_confirmation(
+#'   data,
+#'   by_vars = vars(USUBJID),
+#'   join_vars = vars(AVALC, ADY),
+#'   order = vars(ADY),
+#'   first_cond = AVALC.join %in% c("CR", "PR") & ADY.join - ADY >= 20,
+#'   filter = AVALC == "PR" &
+#'     all(AVALC.join %in% c("CR", "PR", "NE")) &
+#'     count_vals(var = AVALC.join, val = "NE") <= 1 &
+#'     (
+#'       min_cond(
+#'         var = ADY.join,
+#'         cond = AVALC.join == "CR"
+#'       ) > max_cond(var = ADY.join, cond = AVALC.join == "PR") |
+#'         count_vals(var = AVALC.join, val = "CR") == 0
```

Comment:
@bundfussr is the styling right here? why is the min_cond and max_cond brackets done differently? it makes it hard to read

Reply:
Good point, I have updated it.